src/content/structured/components/tree-view/accessibility.mdx

```diff
@@ -0,0 +1,70 @@
+---
+path: "/components/tree-view/accessibility"
+
+navPriority: 41
+
+date: "2024-08-23"
+
+title: "Tree view"
+
+status: "CANARY"
+
+subTitle: "Use the tree view component to display hierarchical data in a tree structure."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/tree-view" },
+    { title: "Code", path: "/components/tree-view/code" },
+    { title: "Accessibility", path: "/components/tree-view/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+The tree view items will focus top-down, including the child nodes of any expanded parent items. Any closed parent nodes will not be included in the focus order.
```

Comment:
Is the second sentence here correct?

Should it be:
```suggestion
The tree view items will focus top-down, including the child nodes of any expanded parent items. Any child nodes of closed parent items will not be included in the focus order.
```
Or maybe just removed because I think it would be expected that child nodes aren't focusable when the parent is collapsed since they won't be visible

Sorry if I've misunderstood :)